source/_dashboards/picture-glance.markdown

```diff
@@ -152,7 +152,71 @@ state_filter:
 entity: switch.decorative_lights
 ```
 
-### Examples
```

Comment:
I would keep `## Examples` as the main title and `### Creating a card to control the camera` as subtitle, together with `### More examples`

Reply:
Good point! Thank you for the suggestion.